src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/typer.ts

```diff
@@ -8,4 +10,25 @@ export interface ITidligereInnvilgetVedtak {
     harTidligereOvergangsstønad: boolean;
     harTidligereBarnetilsyn: boolean;
     harTidligereSkolepenger: boolean;
+    periodeHistorikkOvergangsstønad: IGrunnlagsdataPeriodeHistorikk[];
+}
+
+export interface IGrunnlagsdataPeriodeHistorikk {
+    antallMåneder: number;
+    antallMånederUtenBeløp: number;
+    vedtaksperiodeType: EPeriodetype;
+    fom: string;
+    tom: string;
+}
+
+export interface IStonader {
```

Comment:
Navnet `IStonader` tilsier at dette objektet er en liste som inneholder flere objekter av typen `IStonad`, noe som ikke er tilfellet her. Jeg ville renamet til noe annet, men vet ikke om `IStonad` blir helt riktig, for dette objektet er ikke en Stønad. 

Vi har ellers i løsningen også brukt æ, ø og å, så det går forresten helt fint å brukke disse i props og typenavn :)